### ISSUE TITLE
Removing Empty Neumes and Syllables

### ISSUE THREAD

**Issue by cadagong** [DDMAL/Neon#820]
MEI Actions Dropdown was added to the Neon _gh-pages_ branch. MEI Actions should be tested, particularly the two new features: Removing Empty Neumes and Removing Empty Syllables.

**Comment by JoyfulGen** [comment]
Tested the Remove Empty Syllables function and it works! I'm noticing though that when I click on Remove Empty Syllables, the empty syllables are removed from the MEI, but visually nothing happens. However, as soon as I do any other action in Neon, the changes are applied.

Here's a demo, where I named the empty syllable HALLO and made its (pink) bbox bigger so it's easy to see:

https://user-images.githubusercontent.com/83373378/168618627-80a301c6-73c1-4f05-8537-a273ca3cb86e.mov

Would it be possible to make it so the changes appear as soon as the button is clicked?
